lighthouse-core/report/html/report-styles.css

```diff
@@ -296,27 +298,25 @@
 }
 
 /* Expandable Details (Audit Groups, Audits) */
-
-.lh-expandable-details {
-
-}
-
-.lh-expandable-details__summary {
-  cursor: pointer;
-}
-
 .lh-audit__header {
   display: flex;
   padding: var(--lh-audit-vpadding) var(--text-indent);
+  cursor: pointer;
+}
+
+.lh-audit--load-opportunity .lh-audit__header {
+  display: block;
 }
+
 .lh-audit__header:hover {
   background-color: #F8F9FA;
 }
 
 
 .lh-audit-group[open] > .lh-audit-group__summary > .lh-toggle-arrow,
 .lh-expandable-details[open] > .lh-expandable-details__summary > .lh-toggle-arrow,
-.lh-expandable-details[open] > .lh-expandable-details__summary > div > .lh-toggle-arrow {
+.lh-expandable-details[open] > .lh-expandable-details__summary > div > .lh-toggle-arrow,
+.lh-expandable-details[open] > .lh-expandable-details__summary > div > div > .lh-toggle-arrow {
```

Comment:
the nest grows deeper :( removed in chevron ✨ though?

Reply:
yah in #5137 this gets cleaned up a lot.